Aflores plymouth (#234)
osisoft/Edge-Data-Store-Docs: content/logging/component-logging.md

  - OSIsoft Adapter for Modbus TCP: `http://localhost:5590/api/v1/configuration/Modbus1/Logging`

## Logging schema

The full schema definition for the logging configuration is in the component specific logging file:

  - Edge Data Store: `System_Logging.json`

  - Storage: `Storage_Logging.json`

  - OSIsoft Adapter for OPC UA: `OpcUa1_Logging.json`

  - OSIsoft Adapter for Modbus TCP: `Modbus1_Logging.json`
  

> If you have more than one adapter of the same kind configured, the default file name will incrementally change, for example, `OpcUa2_Logging.json`.

Reviewer: ```suggestion
If you have more than one adapter of the same kind configured, the default file name incrementally changes, for example, `OpcUa2_Logging.json`.
```

| --------------------------- | ---------| --------  | -------- | ----------- |
| **LogFileCountLimit**       | Optional | `integer` | Yes      |  The maximum number of log files that the service will create for the component. It must be a positive integer.  <br> Minimum value: `1` <br> Maximum value: `2147483647` <br> Default value: `31`     |
| **LogFileSizeLimitBytes**   | Optional | `integer` | Yes      | The maximum size in bytes of log files that the service will create for the component. It must be a positive integer.     <br> Minimum value: `1000` <br> Maximum value: `9223372036854775807`  <br> Default value: `34636833`   |
| **LogLevel**                | Optional | reference | No       | The log level settings that you want. The following options are available: <br> `Verbose` - Captures all messages: Verbose, Debug, Information, Warning and Error <br> `Debug` - Captures most messages: Debug, Information, Warning and Error <br> `Information` - Captures most messages: Information, Warning and Error <br> `Warning` - Captures only Warning and Error messages <br> `Error` - Captures Error messages only |

### Log levels

> The logLevel sets the minimum severity for messages to be included in the logs. Messages with a severity below the level set are not included. The log levels in their increasing order of severity are as follows: `Trace`, `Debug`, `Information`, `Warning`, `Error`, `Critical`.

Reviewer: ```suggestion
The `logLevel` sets the minimum severity for messages included in the logs. Messages with a severity below the level set are not included. The log levels in their increasing order of severity are as follows: `Trace`, `Debug`, `Information`, `Warning`, `Error`, `Critical`.
```

The following table has general guidelines for setting the log level.

| **Level**                | **Description**|
|--------------------------|-----------|
| Trace         | Logs that contain the most detailed messages. These messages may contain sensitive application data like actual received values and should not be enabled in a production environment. |

> | Debug | Logs that can be used to troubleshoot data flow issues by recording metrics and detailed flow related information. |

Reviewer: ```suggestion
| Debug | Logs that can be used to troubleshoot data flow issues by recording metrics and detailed flow-related information. |
```
